ethcore/src/client/evm_test_client.rs

```diff
@@ -65,12 +69,21 @@ use ethjson::spec::ForkSpec;
 
 /// Simplified, single-block EVM test client.
 pub struct EvmTestClient<'a> {
-	state: state::State<state_db::StateDB>,
+	state: State<state_db::StateDB>,
 	spec: &'a spec::Spec,
-	dump_state: fn(&state::State<state_db::StateDB>) -> Option<pod_state::PodState>,
+	dump_state: fn(&State<state_db::StateDB>) -> Option<PodState>,
 }
 
-fn no_dump_state(_: &state::State<state_db::StateDB>) -> Option<pod_state::PodState> {
+fn no_dump_state(_: &State<state_db::StateDB>) -> Option<PodState> {
+	None
+}
+
+#[cfg(feature = "to-pod-full")]
+fn dump_state(state: &State<state_db::StateDB>) -> Option<PodState> {
+	state.to_pod_full().ok()
+}
+#[cfg(not(feature = "to-pod-full"))]
```

Comment:
why do we need this feature?

Reply:
It's used in a few places, notably in `evm_test_client` where it's used from the outside in `private-tx`. It can most likely be removed by some light refactoring.

Reply:
removed the feature